src/applications/vaos/tests/project-cheetah/components/ContactInfoPage.unit.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { expect } from 'chai';
+import userEvent from '@testing-library/user-event';
+
+import ContactInfoPage from '../../../project-cheetah/components/ContactInfoPage';
+import { createTestStore, renderWithStoreAndRouter } from '../../mocks/setup';
+import { FETCH_STATUS } from '../../../utils/constants';
+import { cleanup } from 'axe-core';
+
+describe('VAOS <ContactInfoPage>', () => {
+  const initialState = {
+    projectCheetah: {
+      newBooking: {
+        data: {},
```

Comment:
Is this initial state set up required? I was thinking it would use the defaults from the reducer.

Reply:
Good call.  Removed